### ISSUE TITLE
--remote-debug leads to ERROR: connect_to_host: Remote Debugger: Unable to connect. Status: 3.

### ISSUE THREAD

**Issue by mrimvo** [godotengine/godot#48668]
**Godot version:**
3.3

**OS/device including version:**
Ubuntu 18.04

**Issue description:**

When trying to connect to a remote debugger with `--remote-debug`, this error message appears:
```
ERROR: connect_to_host: Remote Debugger: Unable to connect. Status: 3.
   At: scene/debugger/script_debugger_remote.cpp:95.
```

**Steps to reproduce:**
1. start the Godot Editor with an project open
2. in the root folder of the project, start `godot --remote-debug localhost:6007`

EXPECTED: the second godot instance connects to the debugger in the first godot instance
OBSERVED: the game starts after step 2, but doesn't connect to the debugger. Instead the above error message appears

**Minimal reproduction project:**
An empty project with an empty main scene.
[empty-godot-project.zip](https://github.com/godotengine/godot/files/6466549/empty-godot-project.zip)

**Comment by Calinou** [comment]
@mrimvo Does it work if you use `127.0.0.1:6007` instead?

**Comment by mrimvo** [comment]
> 127.0.0.1:6007 instead?
doesn't work, same error message

>Also, are you on a dual-stack IPv4/IPv6 network?
Not sure about this - is there is linux command I can type to check for this? `ifconfig` shows me both an `inet` and `inet6` address for the wifi-adapter. Is this the information you asked for?

> Remember that the Godot editor does not listen for incoming debugger connections. Only the project running from the editor does. See godotengine/godot-proposals#2608.

This sounds like it is exactly the same issue. It looks like the Godot editor doesn't start the debugger to listen on `localhost:6007`, that's why it can't connect with `--remote-debug`.

**Comment by Calinou** [comment]
> This sounds like it is exactly the same issue. It looks like the Godot editor doesn't start the debugger to listen on localhost:6007, that's why it can't connect with --remote-debug.

In this case, I'll close this as a duplicate of https://github.com/godotengine/godot-proposals/issues/2608.
